package.json

```diff
@@ -16,7 +16,9 @@
     "format": "prettier --write './src/**/*.{js,css,md}'"
   },
   "main": "build/bundle.js",
-  "dependencies": {},
+  "dependencies": {
+    "jest-mock": "latest"
```

Comment:
we dont need this... jest already includes the mock function... note that you should add the following to `.eslinrc`
```
globals: {
...
"jest": true
...
}
```

Reply:
i added it as a dependency because the linter was complaining that i had to add `jest-mock` as dependency. See https://github.com/bonlineza/b-lib/pull/26#discussion_r314472783 for the reason why I use that instead of `jest`